.github/workflows/build-swift.yml

```diff
@@ -0,0 +1,143 @@
+---
+name: Build Swift Package
+
+on:
+  push:
+    branches:
+      - "rc"
+      - "hotfix-rc-swift"
```

Comment:
What about master? We currently don't "release" the swift sdk so the pressing concern is to get master snapshots automated since we have to do those manually currently.

Reply:
I was having a discussion around that in slack, in the CloudOps channel. I want to figure this out as well.

For all GitHub release artifacts, those will need to be sourced from the deployment branch. Which is `rc` or `hotifx-rc-swift`. GitHub release artifacts have to come from a tag, and then we create a GitHub release with the artifact attached.

To get `master` builds into a public place to source them in `package.swift`. Those would be stored in an Azure storage account. 

Or if the idea was to release straight from `master` automatically, and have every build be a "GitHub Release" we can go that route for hosting the artifact.

Reply:
We don't want every merge into master to be a new release.

Reply:
My request / feedback in Slack was to get full releases working first with all this, then to look at what `master` artifacts look like (perhaps in parallel). I understand the GitHub release is noisy for every merge and that's not the path, but I want to keep our maintenance burden as low as possible since GitHub Packages doesn't exist for Swift.

Reply:
Ok, so releases from merges into `master` isn't the way we want to go.

Do either of you all like the route of `rc` or `hotfix`/`hotfix-rc` as the release branch, just like how it's currently structured in the workflow?

Reply:
Assuming the need to get artifacts for every master merge is unneeded at the moment, I like the unification of process of using `rc` or similar to prep and ship a release with artifacts.

Reply:
Our requirement **today** is snapshot releases. We don't have any plans on publishing stable releases for the swift SDK for a fairly long time.